### PR TITLE
enhancement: specify custom configuration path

### PR DESCRIPTION
**Pull request by pavelskipenes**
I've tested this with my [docker image](https://hub.docker.com/r/pavelnoen94/github-action-spellcheck) and it seems to be working fine.

### CODE REVIEW

**Review by jonasbn — CHANGES_REQUESTED** [review]
Hi @pavelskipenes 

It looks great, I just require a few minor changes:

1.  Lowercase the input parameter to configpath as recommended in the GitHub Action [documentation](https://docs.github.com/en/free-pro-team@latest/actions/creating-actions/metadata-syntax-for-github-actions#inputs) (yes the GH example uses CamelCase)
2.  The primary style of the GitHub Action YAML is kebab-case, so if you really want to separate the two please use kebab-case like so: "config-path", "configpath" is also acceptable

**Comment by pavelskipenes** [comment]
Thanks for the review @jonasbn. Didn't notice the lower case recommendation on the docs. Thanks for pointing out. I'll choose to use `config-path`.

File: README.md [hunk 1]
```diff
@@ -43,7 +43,25 @@ For example, it could be `.github/workflows/action.yml`
...
+jobs:
+  build:
+    name: Spellcheck
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - uses: rojopolis/spellcheck-github-actions@master
+      name: Spellcheck
+      with:
+        configPath: config/.spellcheck.yml # put path to configuration file here
```
Comment (jonasbn) [inline README.md:61]: We need to adjust the example in the documentation to adhere with the recent name changes

**Comment by pavelskipenes** [comment]
I should have tested before requesting a new review... It seems like `config-path` is converted into `INPUT_CONFIG-PATH` which is not interpreted as a variable in bash (the dash `-` is not recognized as a part of the variable name). I'll have to experiment a bit more to see what I can do. Of course I could just roll back to not using dashes (or using an underscore instead maybe?).

**Review by jonasbn — CHANGES_REQUESTED** [review]
We need to update the documentation as well

**Comment by pavelskipenes** [comment]
It seems like dashes `-` in variables is a valid yaml syntax however problem arrives when actions is trying to convert it to bash variables. According to examples in [this documentation](https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstepswith) they are using underscores in their input parameters. I've also tested bash syntax locally as seen below.
image

There is only one thing that is really weird... In the documentation you mentioned there is a section for [input_id](https://docs.github.com/en/free-pro-team@latest/actions/creating-actions/metadata-syntax-for-github-actions#inputsinput_id)'s which says
> The `<input_id>` must start with a letter or `_` and contain only alphanumeric characters, `-`, or `_`.

That means `config-path` should work just fine but they are not. I'm kinda suspecting that the intended behavior was to convert dashes to underscores but that is not documented there as far as I can see.

I'll just convert everything to underscores for now.

**Review by jonasbn — APPROVED** [review]
Things look good, thanks for the contribution. I will look into a release with your changes shortly

**Comment by jonasbn** [comment]
Hi @pavelskipenes 

You contribution has been included with release 0.9.0, which has just made it to:

- The GitHub Marketplace and DockerHub 

Thanks again for your contribution 

Take care and stay safe

**Pull request merged**